src/components/pages/AdminAddCourse/AdminAddCoursesForm.js

```diff
@@ -1,181 +1,400 @@
+                        <br></br>
+                      </label>
+                    ))
+                  : formValues.days_of_week.forEach(day => (
+                      <label key={`${day}`}>
+                        {`${day}:`}
+                        <br></br>
+                        <TimePicker.RangePicker
+                          defaultValue={[
+                            moment(formValues.start_time, 'HH:mm'),
+                            moment(formValues.end_time, 'HH:mm'),
+                          ]}
+                          onChange={handleTimeChange}
+                          use12Hours={true}
+                          format="HH:mm"
+                          name="course_time"
+                        />
+                        <br></br>
+                      </label>
+                    ))} */}
+              </Form.Item>
+              <Form.Item
+                label={
+                  <label style={{ color: '#096A70', fontSize: '1.1rem' }}>
+                    Location:
+                  </label>
+                }
+                style={{ width: '65%', fontSize: '1.1rem' }}
+                onChange={handleChange}
+              >
+                <Input
+                  value={formValues.course_location}
+                  name="course_location"
+                  style={{ fontSize: '1.1rem' }}
+                />
+              </Form.Item>
+              <Form.Item
+                label={
+                  <label style={{ color: '#096A70', fontSize: '1.1rem' }}>
+                    Maximum Capacity:
+                  </label>
+                }
+                style={{ width: '65%', fontSize: '1.1rem' }}
+                onChange={handleChange}
+              >
+                <InputNumber
+                  value={formValues.max_size}
+                  name="max_size"
+                  style={{ fontSize: '1.1rem' }}
+                />
+              </Form.Item>
+              <Form.Item
+                style={{ width: '65%', fontSize: '1.1rem' }}
+                label={
+                  <label style={{ color: '#096A70', fontSize: '1.1rem' }}>
+                    Minimum Age:
+                  </label>
+                }
+                onChange={handleChange}
+              >
+                <InputNumber
+                  value={formValues.min_age}
+                  name="min_age"
+                  style={{ fontSize: '1.1rem' }}
+                />
+              </Form.Item>
+              <Form.Item
+                style={{ width: '65%', fontSize: '1.1rem' }}
+                label={
+                  <label style={{ color: '#096A70', fontSize: '1.1rem' }}>
+                    Maximum Age:
+                  </label>
+                }
+                onChange={handleChange}
+              >
+                <InputNumber
+                  value={formValues.max_age}
+                  name="max_age"
+                  style={{ fontSize: '1.1rem' }}
+                />
```

Comment:
I believe a .map function can be used to create these three InputNumber fields and clean the code up a little bit.